build_all.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eux
+
+for bp in $(find buildpack -name Dockerfile); do
+	path=$(dirname $bp)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/buildpack:$flavor .) || exit -1
+done
+
+for pb in $(find packagingbuild -name Dockerfile); do
+	path=$(dirname $pb)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingbuild:$flavor .) || exit -1
+done
+
+for pt in $(find packagingtest -name Dockerfile); do
+	path=$(dirname $pt)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingtest:$flavor .) || exit -1
```

Comment:
Won't this create all the packating test ones as "systemd" as the packagingtests are in sub-directories packagingtest/centos7/systemd?
I've found that for packagingtest you need to use docker build -t stackstorm/packagingtest:centos7-systemd etc.

Reply:
Yep, good catch. I was using this script on top of my changes at #101, which removes the `systemd` directories (and moves their contents up one level), so if this is merged after that PR, it should build the images with the correct names.

Currently, however, you're 100% right.

Reply:
Fixed.